mariadb/run.sh

```diff
@@ -26,7 +26,7 @@ else
 
     chown -R mysql:mysql /var/lib/mysql
 
-    mysql_install_db --user=mysql > /dev/null
```

Comment:
Why is "> /dev/null" deleted?

Reply:
I believe it's a good idea to keep the output of `mysql_install_db`.